### ISSUE TITLE
canvasSketch is not a function 

### ISSUE THREAD

**Issue by tatecarson** [mattdesl/canvas-sketch#9]
When trying to run the basic scaffolding code: 
```
const canvasSketch = require('canvas-sketch');

const settings = {
  dimensions: [ 2048, 2048 ]
};

const sketch = () => {
  return ({ context, width, height }) => {
    context.fillStyle = 'white';
    context.fillRect(0, 0, width, height);
  };
};

canvasSketch(sketch, settings);
```

I get this error in Firefox dev 62.0b11: 

```
TypeError: canvasSketch is not a function[Learn More] sketch.js:15:1
[1]<
http://192.168.0.173:9966/sketch.js:15:1
o
http://192.168.0.173:9966/sketch.js:1:257
r
http://192.168.0.173:9966/sketch.js:1:431
<anonymous>
http://192.168.0.173:9966/sketch.js:1:2

```

and in Chrome: 

```
Uncaught TypeError: canvasSketch is not a function
    at Object.1.canvas-sketch (sketch.js? [sm]:14)
    at o (_prelude.js:1)
    at r (_prelude.js:1)
    at _prelude.js:1
```

Node - v8.9.3
npm - 6.1.0
macOS - 10.13.6

Any thoughts?

**Comment by tatecarson** [comment]
Oh! My folder was named `canvas-sketch` and for some reason that broke it. I changed it to `my-sketches` and now it works. 

Thanks for the quick response!

**Comment by mattdesl** [comment]
Ah! Well good catch, that sounds like a bug. :smile:

**Comment by mattdesl** [comment]
This should be fixed in `canvas-sketch-cli@1.0.13` and `canvas-sketch@0.0.15`.
